masterlist.yaml

```
@@ -7805,10 +7805,27 @@ plugins:
         util: 'SSEEdit v4.0.3'
 
   - name: 'tpos_ultimate_esm.esp'
-    url:
-      - link: 'https://www.nexusmods.com/skyrimspecialedition/mods/13196'
-        name: 'The People Of Skyrim Complete SSE'
-    inc: [ 'Holds.esp' ]
+    url: [ 'https://www.nexusmods.com/skyrimspecialedition/mods/13196/' ]
```

Adding the closing slash is fine, but the mod name can stay.
```suggestion
    url:
      - link: 'https://www.nexusmods.com/skyrimspecialedition/mods/13196/'
        name: 'The People Of Skyrim Complete SSE'
```